src/components/anchor-navigation/anchor-navigation-interactions.stories.tsx

```diff
@@ -0,0 +1,80 @@
+// src/components/anchor-navigation/anchor-navigation-interactions.stories.tsx
+import React from "react";
+import { StoryObj, StoryFn } from "@storybook/react";
+import { userEvent, within } from "@storybook/test";
+
+import AnchorNavigation from "./anchor-navigation.component";
+import { DefaultStory } from "./anchor-navigation.stories";
+
+import DefaultDecorator from "../../../.storybook/utils/default-decorator";
+import userInteractionPause from "../../../.storybook/utils/user-interaction-pause";
+import { allowInteractions } from "../../../.storybook/interaction-toggle/reduced-motion";
+
+type Story = StoryObj<typeof AnchorNavigation>;
+
+export default {
+  title: "Anchor Navigation/Interactions",
+  component: AnchorNavigation,
+  parameters: {
+    info: { disable: true },
+    chromatic: { disableSnapshot: true },
+  },
+} as const;
+
+const RenderFromDefault: StoryFn = () =>
+  (DefaultStory as unknown as () => JSX.Element)();
+
+export const NavFocusManagement: Story = {
+  render: RenderFromDefault,
+  play: async () => {
+    if (!allowInteractions()) return;
+
+    await userEvent.tab();
+    await userInteractionPause(50);
```

Comment:
comment: we should avoid using the interactive pause, it uses a node timeout whilst we're testing in the browser. We should be using actual await assertions here instead (eg toHaveFocus)